src/test/java/org/assertj/core/api/Assertions_fail_Test.java

```diff
@@ -37,8 +38,8 @@ public void should_include_message_with_parameters_when_failing() {
   public void should_include_message_with_cause_when_failing() {
```

Comment:
revert these changes, this test is about to check the message not that it an `AssertionFailedError`  was raised.
the same comments for the other tests that were changed